packages/third-parties/src/google/index.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * This is an autogenerated file by update-third-parties.js
+ */
+import React from 'react'
+import Script from 'next/script'
+
+import Base from '../base'
+
+// Embed a Google Maps embed on your webpage
+export function GoogleMapsEmbed(args: any) {
```

Comment:
Can we update the `args: any` to explict props type and rename `Base` component to sth like `ThridPartyScriptEmbed`? This will help remind reviewers and maintainer what's the functionality is